### ISSUE TITLE
New VS code icon is Ugly! 

### ISSUE THREAD

**Issue by fritzlolpro** [Microsoft/vscode#38310]
Hello. Thank you for listening to the herd of morons and makin' childish design mistake.
It's obvious that orange icon was much better than a blue one. 
1. we already have a lot of blue icons, it is really like a plague
2. current blue VScode icon looks COMPLETELY THE SAME as visual studio icon, which is often run at the same time.

![](http://joxi.net/4AkVyaUMexd52q.jpg)

You need to turn back the orange icon or create a new one, which is not the same as the icons of your other products and which is not blue. It would be hard, but I believe you can do it

**Comment by kumarharsh** [comment]
Aahh... I guess no one will be happy with any design vscode icon gets henceforth.

**Comment by fritzlolpro** [comment]
So maybe we need a customize feature? Or at least some collection of ready-to-use ico files?

**Comment by kumarharsh** [comment]
I think you can already do that via resedit or something else. https://superuser.com/questions/48395/resource-editor-for-windows-applications-to-replace-icon-open-source-quick-an

**Comment by excalith** [comment]
TL;DR
There are a few things you can do:
- You can change the cursor manually from windows
- You can change the window name to differ it from other tabs (adding "window.title": "VS Code" to settings)

Funny thing is it was already blue. They changed it into orange and got ugly feedback for orange color as well. They decided to get back to blue again. Here is the journey of the mighty Visual Code icon, just in case if anyone interested: https://code.visualstudio.com/blogs/2017/10/24/theicon

What VS is doing right now is using same logo as a brand for Visual Studio family. Only changing their colors using Split Complementary method (green for insider, blue for visual code, purple for visual studio) for different editions. This allows the icons differ from each other (for VS family) the best, without getting too distinct, like a whole seperate product.

Iit is a matter of design choice actually. But if you are to choose colors to differ from other app icon colors, it is quite impossible to do so since there are lots of different users who use different applications with different colors. You may need to find a new color from a different spectrum in a different dimension.

**Comment by anladwig** [comment]
The orange icon was actually quite nice. With the blue icon, I frequently lose VS code in the sea of purple and blue icons. Maybe just make the icon color a setting and generate the icon based on that setting?

**Comment by drdgvhbh** [comment]
![image](https://user-images.githubusercontent.com/21375952/32806644-3e56563e-c95b-11e7-88e3-17824c1d0c7e.png)
My icon is barely visible when active (vs code window is active). The choice of color should seriously be reconsidered.

When not active it is ok though.
![image](https://user-images.githubusercontent.com/21375952/32806684-6eb09b50-c95b-11e7-9276-48300f68804d.png)

**Comment by AlexLeung** [comment]
I miss the orange icon! It made VS Code easily distinguishable from VS

**Comment by fritzlolpro** [comment]
So, I think that is not a suitable place for discussions with boys who don't understand how design works.

I've found kinda solution, btw.
http://icofx.ro/tutorials/tutorial5.html this is software and tutorial how to replace icon for any application
http://nesamdoom.net/random/orange_code_70x70.ico this is icon, kinda small, but ok

Ofc, this situation is really crappy. MS does not care about users, they do not solve problems, just hiding from them. Blue or orange icons are still bad, but instead of creating strong guidelines for their products, they just change colors.

And if its all about customisation so give users tools for it.

**Comment by kumarharsh** [comment]
But that's exactly it! MS **has** give you tools for it - it's called the Windows OS, which allows tools like icofx and resedit to be created which let you change the icon to whatever you want. Tell me one popular software which lets users choose the icon. Adobe CC? Nope. Office? Nope. Skype? Nope. Atom? Nope. Sublime? Nope. Even gVim? Nope. Firefox? Nope. Chrome? Nope. So what user choice are you talking about?

There was already a long discussion about the icon change here, and this was the final decision. In the end, there are a lot of different people involved with the project, some who liked the new icon, some who liked the old, and some who didn't care much. They had to choose one, and they chose this one. Deal with it.

Also, It'd be better if you take your snark somewhere else. It'd be better for everyone involved. You might also be impinging on the [code of conduct](https://opensource.microsoft.com/codeofconduct/) rules here.

**Comment by bpasero** [comment]
We are currently not planning on changing the icon, please see https://code.visualstudio.com/blogs/2017/10/24/theicon for more information.
